### DISCUSSION TITLE
OpenVK installation guide

### DISCUSSION THREAD

**Discussion by WerySkok** [OpenVK/openvk#59] [Show and tell] [answered]
Другим наверняка будет так же сложно как и мне ставить овк, поэтому я написал детальный гайд. Вдохновлялся доками птеродактиля.
https://gist.github.com/WerySkok/77f9c9ec134e98b0fef2b63655ba13d2

**Comment by ghost** [comment, marked as answer]
Для динозавриков - гайд теперь доступен в документации OVK: https://docs.openvk.su/openvk_engine/centos8_installation/
